src/pe_reports/pe_db/pg_backup.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Set path to save backup files
+path=$PWD
+backup_folder=$path/backups_$(date +%m-%d-%Y)
+
+globalsqlfile=$backup_folder/pedb_globals.sql
+sqlfile=$backup_folder/pedb_dump.sql
+errfile=$backup_folder/stderr.txt
+zipfile=$path/pedb_dump_$(date +%m-%d-%Y).zip
+
+# Create backup folder
+mkdir -p "$backup_folder"
+
+# Create globals backup
+if pg_dumpall --globals-only --no-role-passwords -l "$PE_DB_NAME" -p "$PE_DB_PORT" -U "$PE_DB_USER" -h "$PE_DB_HOST" -w 2> "$errfile" > "$globalsqlfile"; then
+  echo 'Globals dump created'
+else
+  echo 'Globals pg_dump return non-zero code'
+  exit
+fi
+
+# Create backup
+if pg_dump -d "$PE_DB_NAME" -p "$PE_DB_PORT" -U "$PE_DB_USER" -h "$PE_DB_HOST" -Fc -w 2> "$errfile" > "$sqlfile"; then
```

Comment:
Use long flag names:
```suggestion
if pg_dump --database "$PE_DB_NAME" --port "$PE_DB_PORT" --username "$PE_DB_USER" --host "$PE_DB_HOST" --format custom --password 2> "$errfile" > "$sqlfile"; then
```

Reply:
Addressed in [07a282a](https://github.com/cisagov/pe-reports/pull/129/commits/07a282a0ffaa901906905638b182d9c8932f06c3)